frontend/packages/console-app/src/components/quick-starts/QuickStartDrawer.tsx

```diff
@@ -22,26 +31,51 @@ type DispatchProps = {
 type QuickStartDrawerProps = StateProps & DispatchProps;
 
 const QuickStartDrawer: React.FC<QuickStartDrawerProps> = ({
-  isExpanded,
+  children,
   activeQuickStartID,
   onClose,
-  children,
 }) => {
   const quickStart = getQuickStart(activeQuickStartID);
-  // TODO: Add check for tour completed status and send complete alert based on that
-  // const tourCompleteAlert = (
-  //   <Alert variant="success" isInline title="This tour has already been completed" />
-  // );
+
+  const panelContent = quickStart ? (
+    <DrawerPanelContent>
+      <DrawerHead>
+        <div className="co-quick-start-drawer__title">
+          <Title headingLevel="h1" size="xl">
+            {quickStart?.name}
```

Comment:
nit: as in line 40 already checked for `quickStart` so can avoid here
```suggestion
            {quickStart.name}
```